samples/main.cpp

```diff
@@ -42,22 +42,29 @@ std::unique_ptr<NovelRT::Input::BasicInteractionRect> memeInteractionRect;
 
 int main(int argc, char *argv[])
 {
+  std::filesystem::path executableDirPath = NovelRT::Utilities::Misc::getExecutableDirPath();
+  std::filesystem::path resourcesDirPath = executableDirPath / "Resources";
+
+  std::filesystem::path fontsDirPath = resourcesDirPath / "Fonts";
+  std::filesystem::path imagesDirPath = resourcesDirPath / "Images";
+  std::filesystem::path scriptsDirPath = resourcesDirPath / "Scripts";
+
   //setenv("DISPLAY", "localhost:0", true);
   L = luaL_newstate();
   luaL_openlibs(L);
   lua_register(L, "average", average);
-  luaL_dofile(L, "avg.lua");
+  luaL_dofile(L, (scriptsDirPath / "avg.lua").string().c_str());
```

Comment:
We currently include this script as part of `libNovelRT`, but it (in particular) should likely be exclusively an asset to `NovelRTSamples`